source/_patterns/01-atoms/list-group/index.js

```diff
@@ -0,0 +1,17 @@
+// Import tab js from bootstrap
+import { Tab } from 'bootstrap/js/src';
+
+// import $ from 'jquery';
+
+// Module dependencies
+import base from 'base';
+
+// Module styles
+import './_list-group.scss';
+
+export default {
```

Comment:
We don't use this module format any longer. Pull from `epic/v10` for shape of this file.